.gitignore

```diff
@@ -11,7 +11,7 @@ assets/hashes.js
 backend_prod_files/*
 local_compiled_js/*
 core/templates/prod/*
-core/templates/dev/head/dist/*
+webpack/*
```

Comment:
A directory named "webpack" sounds like it contains files for the third-party webpack library.

Should we perhaps rename it to 'dist' or 'prod_bundles' or 'webpack_prod_bundles' or similar? I think I prefer the last one since it gives context that this is "related to Webpack" and "related to prod distribution", but welcome your thoughts.

Perhaps also add a README in that folder explaining what the files in that folder are, and which files they replace (i.e. if we serve those files we can ignore everything in directories XXX).

Reply:
This is also used in dev mode, should we call it `dist` to be consistent with naming used in most webpack projects?

Reply:
Hm, in that case, I think I would lean towards webpack_bundles (with a README file). My interpretation of "dist" is that it's usually for production stuff (it's the software "distribution"), and the name is also probably not that helpful for newer contributors. WDYT?

Reply:
Renamed to webpack_bundles.